### ISSUE TITLE
pretrained model 

### ISSUE THREAD

**Issue by KeyBoarder8** [ictnlp/DST#1]
Thanks for open source. Some questions:
Should pretrained model in phase 1(Full-Sentence Pre-training) be a finetuned model for phase 2(Simultaneous Translation Fine-Tuning). For example, use a parameter “--restore-file” in fairseq to train the model phrase 2?

**Comment by ShoutaoGuo** [comment]
Yes, the model of the first state is the begining of the second state
